pkg/slo-controller/config/config.go

```diff
@@ -189,20 +189,23 @@ func GetNodeColocationStrategy(cfg *ColocationCfg, node *corev1.Node) *Colocatio
 
 	nodeLabels := labels.Set(node.Labels)
 	for _, nodeCfg := range cfg.NodeConfigs {
+		if nodeCfg.NodeSelector == nil {
```

Comment:
A nil NodeSelector is translated to match nothing via `LabelSelectorAsSelector()`. Maybe we do not need to check if it is nil.

Reply:
en,indeed.